test/integration/test_redmine_backup_restore.py

```diff
@@ -0,0 +1,107 @@
+import subprocess, os, re, shutil
+from selenium import webdriver
+from pyvirtualdisplay import Display
+
+def wrong_dir_error():
+    print "ERROR\tTest ran from wrong directory!"
+    print "\tRun test from top level directory, outside of test folder"
+    exit(1)
+
+def check_path_test_dir(current_dir):
+    for x in ["test","integration"]:
+        if x in current_dir:
+            wrong_dir_error()
+
+def log_into_ci():
+    global DRIVER, DISPLAY
+    DISPLAY = Display(visible=0)
+    DISPLAY.start()
+    DRIVER = webdriver.Firefox() # initilize selenium
+    DRIVER.get("http://admin:passwd@localhost/redmine")
+
+def log_into_redmine():
+    DRIVER.get("http://localhost/redmine/login")
+    element = DRIVER.find_element_by_id("username")
+    element.send_keys("admin")
+    element = DRIVER.find_element_by_id("password")
+    element.send_keys("admin")
+    DRIVER.find_element_by_name("login").click()
+
+def create_redmine_project():
+    DRIVER.get("http://localhost/redmine/projects")
+    DRIVER.get("http://localhost/redmine/projects/new")
+    DRIVER.find_element_by_id("project_name").send_keys("Test Project")
+    DRIVER.find_element_by_name("commit").click()
+
+
+def create_redmine_issue():
+    DRIVER.get("http://localhost/redmine/projects/test-project/issues/new")
+    DRIVER.find_element_by_id("issue_subject").send_keys("Test Issue of Type Bug")
+    DRIVER.find_element_by_name("commit").click()
+    DRIVER.close()
+
+
+def setup_function(function):
+    current_dir = os.getcwd() # get current directory
+    check_path_test_dir(current_dir) # ensure we aren't in the test directory
+    test_dir = os.path.abspath(os.path.join(current_dir, "test/integration")) # get test directory path
```

Comment:
os,path.join works with multiple args. The hard-coded '/' defeats the purpose of using join.